core/templates/dev/head/pages/exploration-editor-page/improvements-tab/feedback-improvement-task/feedback-improvement-task.directive.ts

```diff
@@ -65,6 +65,12 @@ angular.module('oppia').directive('feedbackImprovementTask', [
             return DateTimeFormatService.getLocaleAbbreviatedDatetimeString(
               $scope.getLatestMessage().updatedOn);
           };
+
+          $scope.wasUpdatedToday = function() {
+            var millisUptillToday = new Date().setHours(0, 0, 0, 0);
+            return (
+              $scope.getLatestMessage().updatedOn - millisUptillToday >= 0 );
```

Comment:
Sorry, a couple more small edits:

- "uptill" is not a word. Let's change it to millisUntilToday or, perhaps better, msecsUntilToday (since "milli" is a general prefix that can also represent things like millimeters, etc.)
- You can simplify the inequality: $scope.getLatestMessage().updatedOn >= msecsUntilToday.

Reply:
Okay, will make the changes.